### PR TITLE
add gpu jobs for 1.9

### PR DESCRIPTION
**Pull request by BenTheElder**
/area jobs

### CODE REVIEW

**Comment by BenTheElder** [comment]
xref: #5558 
/cc @mindprince 
This is not done, I need to check back over this and add the jobs to testgrid but I'm firefighting an issue with the docker graph storage for the dind jobs right now.

-beta is 1.9 until release

File: jobs/config.json [hunk 1]
```diff
@@ -6151,7 +6189,7 @@
     "args": [
       "--check-leaked-resources",
       "--deployment=gke",
-      "--extract=ci/k8s-stable1",
+      "--extract=ci/k8s-beta",
```
Comment (BenTheElder) [inline jobs/config.json]: this is wrong, it should be swapped with the job above

File: testgrid/config/config.yaml [hunk 2]
```diff
@@ -3201,6 +3226,10 @@ dashboards:
     test_group_name: ci-kubernetes-e2e-gce-device-plugin-gpu-stable1
     alert_options:
       alert_mail_to_addresses: 'gke-kubernetes-accelerators-bugs@google.com'
+  - name: gce-device-plugin-gpu-1-9
```
Comment (mindprince) [inline testgrid/config/config.yaml]: All the tests should be listed in this `wg-resource-management` dashboard.

Comment (BenTheElder) [reply testgrid/config/config.yaml]: done, thanks

**Comment by mindprince** [comment]
Looks like we don't have enough GPU projects in boskos now. Can you add a few more projects. You will have to make sure they have APIs enabled for both GCE and GKE. And request quota for 8 NVIDIA_TESLA_P100 and 8 NVIDIA_TESLA_K80 is us-west1.

**Comment by BenTheElder** [comment]
@krzyzacy can you handle adding some GPU projects? I see how to add them to boskos but not how exactly they should be configured prior to adding them to boskos.

**Comment by BenTheElder** [comment]
Discussed with Sen, adding:
`k8s-jkns-e2e-gce-gpus-beta`
`k8s-jkns-e2e-gke-gpus-beta`

**Comment by krzyzacy** [comment]
add them to https://github.com/kubernetes/test-infra/blob/master/boskos/resources.json#L463-L476

**Comment by BenTheElder** [comment]
Associated CL(s) and quota requests sent. I will add these to boskos once they're done.

**Pull request merged**